### ISSUE TITLE
Model overfitting question

### ISSUE THREAD

**Issue by HardeyPandya** [haoxiangsnr/IRM-based-Speech-Enhancement-using-LSTM#5]
I am just very beginner in this area and not much familiar with Deep Learning standard procedures and all.

Actually, I have converted this code into time domain and then used TCNN model architecture code https://github.com/LXP-Never/TCNN for this. I am using google Colab platform for this.

I have used TIMIT dataset which is having 8732 utterances and randomly mixed with UrbanSound8K noises at -5dB,-4dB,-3dB,-2dB,-1dB,0dB and 1dB, so I am having 8732 noisy speeches. Then I convert it into overlapping frames. And output proceeds.

But I am not sure whether it is overfitting - After 600 epochs of training validation, average PESQ score obtained is 2.13. Average PESQ between clean speech and noisy speech of UrbanSound8k + TIMIT clean speech is around 1.8 .
(On each epoch 900 noisy utterences are trained and on next epoch utterences are shuffled and it is trained on other 900 utterences)

But upon testing, I use NOIZEUS database which is unseen to the TCNN network. I am getting very low PESQ score of 1.42 after loading checkpoints. Also when i run same inference script I get different PESQ scores like 1.3, 1.4 or 1.5 something like that ! On same model checkpoint.

Any suggestions why this is the case ? It would be very helpful.

Thanks.
